Library/Homebrew/extend/os/mac/software_spec.rb

```diff
@@ -4,22 +4,28 @@ class SoftwareSpec
   undef uses_from_macos
 
   def uses_from_macos(deps, **args)
+    @macos_deps ||= []
```

Comment:
Thoughts on just `uses_from_macos`? I think the use of `dips` here is a bit misleading because at this stage this aren't actually dependencies.